packages/block-library/src/media-text/media-container.native.js

```diff
@@ -222,7 +221,7 @@ class MediaContainer extends Component {
 		);
 	}
 
-	renderVideo( params, openMediaOptions ) {
+	renderVideo( params ) {
```

Comment:
Since this function no longer has the `openMediaOptions` parameter, we could also remove it from the following call:
https://github.com/WordPress/gutenberg/blob/8626035c226667df7895c43cd3c1ce148081f894/packages/block-library/src/media-text/media-container.native.js#L306

Reply:
Nice catch, updated in https://github.com/WordPress/gutenberg/pull/40651/commits/192f99dae7d512a98b22cbe4158429832c167e70